### PR TITLE
feat: add purple bubble keywords

### PR DESCRIPTION
**Pull request by kcoderhtml**
Added the new purple bubble emoji from slack to the keywords list.

### CODE REVIEW

File: src/lib/emojiKeywords.js [hunk 1]
```diff
@@ -236,5 +236,7 @@ export default {
   "orpheus podcast": "tos-icon",
   podcast: "studio_microphone",
   quest: "quests",
-  puzzmo: "puzzmo"
+  puzzmo: "puzzmo",
+  "purple bubble": "purplebubble",
+  purplebubble:purplebubble
```
Comment (JosiasAurel) [inline src/lib/emojiKeywords.js:241]: i think you should put the value of 'purplebubble' within quotes

**Review by JosiasAurel — APPROVED** [review]

File: src/lib/emojiKeywords.js (hunk 1 above)
Comment (JosiasAurel) [inline src/lib/emojiKeywords.js:241]: i think you should put the value of 'purplebubble' within quotes

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

File: src/lib/emojiKeywords.js (hunk 1 above)
Comment (kcoderhtml) [reply src/lib/emojiKeywords.js:241]: got it! thanks!

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Pull request merged**